docs/integrations/xsoar-ide.md

```diff
@@ -0,0 +1,65 @@
+---
+id: xsoar-ide
+title: Cortex XSOAR IDE
+---
+Cortex XSOAR offers a built-in platform IDE which is available through the product's web UI. 
+
+:::note Pro Tip
+We recommend to try out the [PyCharm IDE Plugin](pycharm-plugin). It simplifies third-party integration and script development by enabling users to author Python content for Cortex XSOAR directly in PyCharm. Highly recommended for developing [Python Unit Tests](unit-testing). 
+:::
+
+
+It is important to take a moment and familiarize yourself with the Cortex XSOAR IDE as it may work differently than other IDEs you are familiar with. Some key differences between the Cortex XSOAR IDE and, for example PyCharm, is the absence of an interpreter.
+
+## The Script Helper
+Cortex XSOAR is equipped with a script helper which is accessible via the button below:
+
+![](../doc_imgs/integrations/script-helper.png)
```

Comment:
Do we want image alt tags for SEO purposes, or are they out of scope?

Reply:
not that important